repos/spack_repo/builtin/packages/pareval/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.generic import Package
+from spack_repo.builtin.build_systems.rocm import ROCmPackage
+
+from spack.package import *
+
+
+class Pareval(Package, CudaPackage, ROCmPackage):
+    """ParEval benchmark for evaluating LLMs' parallel code generation capabilities"""
+
+    homepage = "https://github.com/parallelcodefoundry/pareval"
+    url = "https://github.com/parallelcodefoundry/pareval"
+    git = "https://github.com/parallelcodefoundry/pareval.git"
+
+    maintainers("Dando18")
+
+    license("MIT", checked_by="Dando18")
+
+    version("develop", branch="develop")
+    version("1.2", tag="v1.2")
+    version("1.1", tag="v1.1")
+    version("1.0", tag="v1.0")
```

Comment:
Correcting the URL property yields the following sha256s:

```suggestion
    version("1.2", sha256="8b374afc07bc177a3ba335ff1fe782b3a1943bbcd2dd7ac0a62dedfa4f44f651")
    version("1.1", sha256="c07bf9b1027afed97281eb2fbcbe8f3d819ac12045ba9d1a7a90d5488f92cc53")
    version("1.0", sha256="314cf62e421c9382f4344d7da57f3dcdf9ab3e1272afd6a7badabf0ffe80f294")
```

after running `spack checksum pareval`.